android/build.gradle

```diff
@@ -14,8 +14,14 @@ android {
   }
 }
 
+allprojects {
+  repositories {
+    jcenter()
+    maven { url "$projectDir/../../react-native/android" }
```

Comment:
@yorkie react-native  android hasn't published to maven , dependency by local dir.